src/utils/base64.ts

```diff
@@ -0,0 +1,21 @@
+import { btoa, atob } from 'abab';
+import { JOSEError } from './errors';
+
+export function encodeBase64(value: string): string {
+  let encoded = btoa(value);
+  if (encoded) {
+    // eslint-disable-next-line no-div-regex
+    return encoded.replace(/=/g, '').replace(/\+/g, '-').replace(/\//g, '_');
```

Comment:
Not clear for me why we need to use replace regex rule here.

Reply:
By default, the `replace()` method replaces the first match if the regexp doesn’t use the global flag (`g`). To replace all matches, you use the global flag (`g`) in the regexp.

Reply:
Not clear to me is why we don't use encoded directly instead refer replace.

Reply:
Yep it should be a good idea move your reply in a comment.

Reply:
done!